Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mozillamarketplace/centos-mysql-mkt:0.2
+
+RUN yum install -y supervisor
+
+ENV IS_DOCKER 1
+
+RUN mkdir -p /pip/{cache,build}
+ADD requirements /pip/requirements
+WORKDIR /pip
+RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/
```

Comment:
I still don't know why peep wasn't a patch submitted to pip :disappointed: but I'll check it out